docs/general/developer-resources/emerald-paratime/README.mdx

```diff
@@ -36,6 +36,30 @@ To get started building on our Emerald ParaTime, you can use our public Web3 gat
   * Decimal: 42262
 * Block explorer: [https://explorer.emerald.oasis.dev](https://explorer.emerald.oasis.dev)
 
+<button
+  class="button button--secondary button--outline"
+  onClick={() =>
+    window.ethereum.request({
+      method: 'wallet_addEthereumChain',
+      params: [
+        {
+          chainId: '0xa516',
+          chainName: 'Emerald Paratime Mainnet',
+          nativeCurrency: {
+            name: 'Emerald Rose',
+            symbol: 'ROSE',
+            decimals: 18,
+          },
+          rpcUrls: ['https://emerald.oasis.dev', 'wss://emerald.oasis.dev/ws'],
+          blockExplorerUrls: ['https://explorer.emerald.oasis.dev'],
+        },
+      ],
+    })
+  }
+>
+  Add Emerald Mainnet to MetaMask or Brave Wallet
```

Comment:
I would put it more explicit. Same for testnet.

```suggestion
  Click here to add Emerald Mainnet to MetaMask or Brave Wallet
```

Reply:
Even better, could you move the button to the top just below the Mainnet subtitle and change the label to

 ```suggestion
  Click here to register Emerald Mainnet at MetaMask or Brave Wallet automatically
```

So folks see the button before they start reading the actual chain ids, endpoints etc. (and already on the way to filling metamask fields).

Reply:
New preview

![image](https://user-images.githubusercontent.com/3758846/164056516-139f6881-a18d-4bd4-9335-a94fabadf319.png)

Reply:
How about: 
![image](https://user-images.githubusercontent.com/3758846/164069799-3f67fb42-94c7-476f-92c4-ad8e250f9870.png)